src/platform/wifi/nm-wifi-utils-nl80211.c

```diff
@@ -40,9 +40,13 @@
 #define _NMLOG_PREFIX_NAME      "wifi-nl80211"
 #define _NMLOG(level, domain, ...) \
 	G_STMT_START { \
-		nm_log ((level), (domain), NULL, NULL, \
-		        "%s: " _NM_UTILS_MACRO_FIRST(__VA_ARGS__), \
-		        _NMLOG_PREFIX_NAME \
+		char _ifname_buf[IFNAMSIZ]; \
+		const char *_ifname = nl80211 ? nmp_utils_if_indextoname (nl80211->parent.ifindex, _ifname_buf) : NULL; \
```

Comment:
in all(?) other `_NMLOG()` implementations, this `nl80211` variable is commonly called `self`.

Reply:
Yes, but not here...

Reply:
Sorry for being unclear. Could you rename the "nl80211" variables to "self", which is what we do almost everywhere else?

Reply:
Okay...